### DISCUSSION TITLE
Cannot find latest edc :: vault-aws in Maven Central Repository

### DISCUSSION THREAD

**Discussion by chlorochrule** [eclipse-edc/Connector#4622] [Q&A] [answered]
Now, the latest release version of [Technology-Aws](https://github.com/eclipse-edc/Technology-Aws) repository is `0.10.1`. However I can't find that version of artifacts. The latest version of published in Maven Central Repository is `0.7.0`.
So, please let me know where I should refer.

https://mvnrepository.com/artifact/org.eclipse.edc/vault-aws

**Comment by wolf4ood** [comment, marked as answer]
https://central.sonatype.com/artifact/org.eclipse.edc.aws/vault-aws

**Comment by chlorochrule** [comment]
@wolf4ood Thanks a lot!

```diff
- edc-vault-aws = { module = "org.eclipse.edc:vault-aws", version.ref = "0.10.1" }
+ edc-vault-aws = { module = "org.eclipse.edc.aws:vault-aws", version.ref = "0.10.1" }
```

Worked!
